nui-libgdx/src/main/java/org/terasology/nui/backends/libgdx/NUIInputProcessor.java

```diff
@@ -123,9 +131,7 @@ public boolean keyTyped(char character) {
         if (character == '\\') {
             lastKey = Keyboard.Key.BACKSLASH;
         }
-
-        keyboardActionQueue.add(new KeyboardAction(lastKey, ButtonState.DOWN, character));
-        keyboardActionQueue.add(new KeyboardAction(lastKey, ButtonState.UP, character));
+        keyboardCharQueue.add(new CharKeyboardAction(character));
```

Comment:
What has changed here? Did you we do some kind of _key code escaping_ before which is now obsolete because raw input and char input have been split up?

Reply:
yeah. KeyUp and KeyDown moved into RawKeyboard.
need some investigate for LibGDX. ( nui v1.0 not used by and LibGDX's game :) )